src/app/Traits/ExistingUser.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace NetworkRailBusinessSystems\UserLogin\Traits;
+
+interface ExistingUser
```

Comment:
Either expand your readme, or add a docblock to explain the purpose of this.

The brief section of the readme does not explain what the intention is.